### DISCUSSION TITLE
Upgrade Mastercontainer's PHP from version 8.2.8 to 8.2.9

### DISCUSSION THREAD

**Discussion by coderInCape** [nextcloud/all-in-one#3306] [Questions] [answered]
Hi All, 
Is there any plans for upgrading the mastercontainer's (port 8080) PHP from version 8.2.8 to 8.2.9?

Regards.

**Comment by Zoey2936** [comment, marked as answer]
v8.2.10 will be inluced in the next release: https://github.com/nextcloud/all-in-one/pull/3301

**Reply by coderInCape** [reply]
Thanks Zoey. 
When is it gonna be released, if I may ask?

**Reply by Zoey2936** [reply]
if everything work arround 12.09. https://github.com/nextcloud/all-in-one/issues/3299

**Reply by coderInCape** [reply]
Thanks Zoey.
